sympy/utilities/runtests.py

```diff
@@ -662,6 +662,7 @@ def _doctest(*paths, **kwargs):
     split  = kwargs.get('split', None)
     blacklist.extend([
         "doc/src/modules/plotting.rst",  # generates live plots
+        "doc/src/modules/physics/mechanics/autolev_parser.rst",
```

Comment:
Why is this blacklisted?

Reply:
The doctest in autolev_parser.rst is not meant to be executed. It just shows the user what commands should be executed in the shell.